test/nodejs/locations.js

```diff
@@ -0,0 +1,8 @@
+// Somewhere in Monaco ..
+// http://www.openstreetmap.org/#map=18/43.73185/7.41772
```

Comment:
Better name would be `constants` to keep it more general. Might be worth merging this with `osrm-data-path`.